packages/hoppscotch-common/src/helpers/import-export/import/openapi.ts

```diff
@@ -754,19 +754,21 @@ const parseOpenAPIUrl = (
    **/
 
   if (objectHasProperty(doc, "swagger")) {
-    const host = doc.host?.trim() || "<<baseUrl>>"
-    const basePath = doc.basePath?.trim() || ""
-    return `${host}${basePath}`
+    // TODO: dynamically add doc.host, doc.basePath value as variables in the environment if available. or notify user to add it.
+    // add base url variable to each request
+    const hostVar = "<<baseUrl>>"
+    const basePathVar = doc.basePath ? "<<basePath>>" : ""
```

Comment:
We can keep the existing behaviour until the support for environment variable addition is brought.